npm/cmd/main.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2018 Microsoft. All rights reserved.
+// MIT License
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+
+	"github.com/Azure/azure-container-networking/log"
+	npmconfig "github.com/Azure/azure-container-networking/npm/config"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"k8s.io/klog"
+)
+
+// Version is populated by make during build.
+var version string
+var cfgFile string
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
```

Comment:
very subjective, but i prefer to put the root command in a `root.go`, and keep `main.go` very lean, with only a `main()` and the min cobra imports